creator-node/src/routes/files.js

```diff
@@ -457,52 +348,76 @@ const getCID = async (req, res) => {
    * 2. If retrieved, stream from file system
    * 3. Else, error
    */
-  const blockStartMs = Date.now()
   try {
-    startMs = Date.now()
-    const found = await findCIDInNetwork(storagePath, CID, req.logger, trackId)
+    const found = await findCIDInNetwork(
+      storagePath,
+      CID,
+      req.logger,
+      libs,
```

Comment:
prob bad master merge but libs param doesn't exist on this function, so this is actually broken now 😬